src/ray/core_worker/core_worker_test.cc

```diff
@@ -299,8 +299,43 @@ TEST_F(ZeroNodeTest, TestWorkerContext) {
   ASSERT_EQ(context.GetNextPutIndex(), 3);
 }
 
+TEST_F(ZeroNodeTest, TestActorHandle) {
+  ActorHandle handle1(ActorID::FromRandom(), ActorHandleID::FromRandom(),
+                      ray::rpc::Language::JAVA,
+                      {"org.ray.exampleClass", "exampleMethod", "exampleSignature"});
+
+  auto forkedHandle1 = handle1.Fork();
+  ASSERT_EQ(1, handle1.NumForks());
+  ASSERT_EQ(handle1.ActorID(), forkedHandle1->ActorID());
+  ASSERT_NE(handle1.ActorHandleID(), forkedHandle1->ActorHandleID());
+  ASSERT_EQ(handle1.ActorLanguage(), forkedHandle1->ActorLanguage());
+  RAY_LOG(INFO) << handle1.ActorCreationTaskFunctionDescriptor().size();
+  RAY_LOG(INFO) << forkedHandle1->ActorCreationTaskFunctionDescriptor().size();
```

Comment:
Are these logs useful?  I guess they could be removed, right?

Reply:
My bad.